src/uu/tail/src/tail.rs

```diff
@@ -483,9 +483,12 @@ fn unbounded_tail<T: Read>(reader: &mut BufReader<T>, settings: &Settings) -> UR
     // contains count lines/chars. When reaching the end of file, output the
     // data in the ringbuf.
     match settings.mode {
-        FilterMode::Lines(count, _) => {
-            for line in unbounded_tail_collect(lines(reader), count, settings.beginning) {
-                print!("{}", line);
+        FilterMode::Lines(count, sep) => {
+            for line in unbounded_tail_collect(lines(reader, sep), count, settings.beginning) {
+                match std::str::from_utf8(&line) {
```

Comment:
Could we just write the bytes directly to stdout without converting to a `&str`? Something like:
```rust
stdout().write(line);
```
That's probably what the GNU does.

Reply:
Yes, I'll make that change.